examples/secaggplus-mt/settings.py

```diff
@@ -0,0 +1,3 @@
+LOG_EXPLAIN = True
+
+NUM_ROUNDS = 3
```

Comment:
We can probably hard-code these in the respective modules and get rid of this one, just to simplify things a bit

Reply:
sure, fixed it.